src/output.cpp

```diff
@@ -194,7 +199,9 @@ static void HandleErrorOutput(const std::string& err) {
 
 	Input::ResetKeys();
 	while (!Input::IsAnyPressed()) {
-		DisplayUi->Sleep(1);
+#if !defined(EMSCRIPTEN) && !defined(USE_LIBRETRO)
+		Game_Clock::SleepFor(1ms);
```

Comment:
Actually I can't remember anymore what the purpose of the sleep was.

Reply:
I guess was to loosen stress on this tight loop.

Reply:
yeah, that should be the reason :+1:

Reply:
Will revisit this when I rebased the PR and add comments

Reply:
This is adjusted for current state, but full solution punted to #2075